test/module/irohad/common/validators_config.hpp

```diff
@@ -19,6 +19,12 @@ namespace iroha {
             std::make_shared<shared_model::validation::ValidatorsConfig>(
                 getTestsMaxBatchSize()));
 
+    static const std::shared_ptr<shared_model::validation::ValidatorsConfig>
+        kTestsValidatorsConfigNoDuplicateTolerance(
```

Comment:
That kind of taxonomy does not look sound.
Do we have tests that require allowed transactions duplicates for validators?

Reply:
I guess those are only proposal-related tests again. If so, then it good to have `false` as default and if needed - create something like `kProposalTestsValidatorsConfig` instead.

Reply:
Fixed